nautobot_design_builder/signals.py

```diff
@@ -49,7 +49,7 @@ def create_design_instance_statuses(**kwargs):
         "Pending": ColorChoices.COLOR_ORANGE,
         "Rolled back": ColorChoices.COLOR_RED,
     }
-    for _, status_name in chain(choices.DesignInstanceStatusChoices, choices.DesignInstanceLiveStateChoices):
+    for _, status_name in chain(choices.DeploymentStatusChoices):
```

Comment:
you have to create the "Uknown" status that is defined in the choices in the previous mapping. Also, remove the non used ones